tests/pyfrc_test.py

```diff
@@ -0,0 +1,6 @@
+'''
```

Comment:
This should be another PR -- we'll also want some tests, not just the boilerplate.